src/main/java/org/jenkinsci/plugins/docker/traceability/core/DockerTraceabilityRootAction.java

```diff
@@ -443,21 +445,23 @@ public HttpResponse doQueryContainer(
             }
         }
         
-        // Return raw JSON in the response
-        return toJSONResponse(result);
+        return toFormattedResponse(result, format);
     }  
     
     /**
      * Retrieves the latest raw status via API.
-     * The output will be retrieved in JSON.
+     *
      * @param id ID of the image, for which the info should be retrieved.
      *    Short container IDs are not supported.
+     * @param format Format used in the output response.
+     *
      * @throws IOException Processing error
      * @throws ServletException Servlet error
-     * @return {@link HttpResponse}
+     *
+     * @return Response available in different formats {@link HttpResponse}
```

Comment:
The resulting text will be "Response available in different formats HttpResponse". The last word is not consistent with the sentence. "HttpResponse available in different formats" would be better